src/main/java/ladder/controller/LadderController.java

```diff
@@ -11,10 +13,18 @@ public static void run() {
         String nameString = InputView.nameOfPlayer();
         NameList nameList = new NameList(nameString);
 
-        int heightOfLadder = StringToIntConverter.toInt(InputView.heightOfLadder());
+        String resultString = InputView.resultOfGame();
+        GameResultList gameResultList = new GameResultList(resultString, nameList.numOfName());
 
+        int heightOfLadder = StringToIntConverter.toInt(InputView.heightOfLadder());
         Ladder ladder = new Ladder(heightOfLadder, nameList.numOfName());
 
-        OutputView.printLadderGameResult(nameList, ladder);
+        GameResult gameResult = new GameResult(nameList, gameResultList);
+        OutputView.printLadderGameResult(nameList, ladder, gameResultList);
+
+        gameResult.ladderGame(ladder);
+
+        String resultPlayer = InputView.printWantToKnowResult();
+        OutputView.printResultOfGame(gameResult,resultPlayer);
```

Comment:
```suggestion
        OutputView.printResultOfGame(gameResult, resultPlayer);
```
공백 컨벤션이 지켜지지 않은 곳이 적지 않게 있네요
커밋 전에 확인해 보시면 좋겠습니다